### PR TITLE
Creation component articles

### PR DESCRIPTION
**Pull request by SelinGitt**

### CODE REVIEW

**Review by Guillaum6e — APPROVED** [review]

**Review by Guillaum6e — DISMISSED** [review]

**Review by Guillaum6e — APPROVED** [review]

File: src/app/article.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Category } from "./category"
+import { User } from "./user"
+
+export interface Article {
+
+    id: number
+    title: string
```
Comment (Benja-san) [inline src/app/article.ts:7]: add slug parameter

File: src/app/category.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export interface Category {
+    id: number
+    name: string
```
Comment (Benja-san) [inline src/app/category.ts:3]: add slug parameter

File: src/app/components/articles/article/body/body.component.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  background-color: $main-background-color;
+  color: $main-text-color;
+  padding-left: $big-border; 
+  padding-right: $big-border;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+  h2 {
```
Comment (Benja-san) [inline src/app/components/articles/article/body/body.component.scss:13]: wrap all those tag targets inside the .body class

File: src/app/components/articles/article/other/other.component.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+@use "/src/styles.scss" as *;
+
+.other {
+    margin-right: $big-border;
+}
+
+h2 {
```
Comment (Benja-san) [inline src/app/components/articles/article/other/other.component.scss:7]: wrap h2 into other class for better scss

File: src/app/components/articles/article/title/title.component.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<section class="title">
+    <div class="article">
+        <h2>{{ article.title }}</h2>
```
Comment (Benja-san) [inline src/app/components/articles/article/title/title.component.html:3]: h1 ?

File: src/app/components/articles/article/title/title.component.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+@Component({
+  selector: 'app-article-title',
+  templateUrl: './title.component.html',
+  styleUrls: ['./title.component.scss']
+})
+export class TitleComponent {
+  article: Article = {
+    id: 1,
+    title: "Le caramel qui colle aux dents : pour ou contre ?",
```
Comment (Benja-san) [inline src/app/components/articles/article/title/title.component.ts:12]: add slug entry

File: src/app/components/articles/article/article.component.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
+@use "/src/styles.scss" as *;
+
+* {
```
Comment (Benja-san) [inline src/app/components/articles/article/article.component.scss:3]: remove this part since it has been integrated in global scss file

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/article.ts (hunk 1 above)
Comment (FlorianHe) [reply src/app/article.ts:7]: Slug will be intergrated will using the datas from US-Interfaces

File: src/app/category.ts (hunk 2 above)
Comment (FlorianHe) [reply src/app/category.ts:3]: Slug will be intergrated will using the datas from US-Interfaces

File: src/app/components/articles/article/title/title.component.ts (hunk 6 above)
Comment (FlorianHe) [reply src/app/components/articles/article/title/title.component.ts:12]: Slug will be intergrated will using the datas from US-Interfaces

File: src/app/components/articles/article/body/body.component.scss (hunk 3 above)
Comment (SelinGitt) [reply src/app/components/articles/article/body/body.component.scss:13]: Done.

File: src/app/components/articles/article/other/other.component.scss (hunk 4 above)
Comment (SelinGitt) [reply src/app/components/articles/article/other/other.component.scss:7]: Done.

File: src/app/components/articles/article/title/title.component.html (hunk 5 above)
Comment (SelinGitt) [reply src/app/components/articles/article/title/title.component.html:3]: Done.

File: src/app/components/articles/article/article.component.scss (hunk 7 above)
Comment (SelinGitt) [reply src/app/components/articles/article/article.component.scss:3]: Done.

**Review by Guillaum6e — APPROVED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**